e2e/helpers/JSWait.js

```diff
@@ -0,0 +1,15 @@
+const {getAccessibilityTestResult} = require('./accessibility/runner');
+const {generateAccessibilityReport} = require('../tests/reporter/customReporter');
+const testConfig = require('../config.js');
+
+class JSWait extends Helper {
```

Comment:
not sure why it is called JSWait, for me looks misleading, as it does not wait, but generate report

Reply:
This was called this way because in other projects the function to generate the report is added to this helper, I can change the name or this file can be kept this way if later on a wait is added to the tests, let me know what's best.

Reply:
please rename it, we dont need wait in our tests. Or if you want you can move method to hooks_helper.js. In this helpers we keep codecept lifecycle callbacks